### ISSUE TITLE
GPT-4-Vision Upload images or files

### ISSUE THREAD

**Issue by Ink-Osier** [yGuy/chatgpt-mattermost-bot#73]
Does this service support carrying images uploaded in mattermost to ask questions using gpt-4-vision? Based on my testing it doesn't seem to work?

**Comment by yGuy** [comment]
No, it doesn't. Only text is sent to the LLM, which can answer with text and images. PRs to add more features are welcome.
